File: app/src/main/java/com/test/quizexampleinterview/authentication/AuthModel.java
package com.test.quizexampleinterview.authentication;

public class AuthModel {

    private String email;
    private String password;

    public AuthModel(String email) {
        this.email = email;
    }

    public AuthModel(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }

}

File: app/src/main/java/com/test/quizexampleinterview/authentication/AuthModel.kt
package com.test.quizexampleinterview.authentication

class AuthModel {
    var email: String
        private set
    var password: String? = null
        private set

    constructor(email: String) {
        this.email = email
    }

    constructor(email: String, password: String?) {
        this.email = email
        this.password = password
    }

}